timeutils/month.go

```diff
@@ -0,0 +1,102 @@
+package timeutils
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+)
+
+const earliestYear = 1801
+const latestYear = 2999
+
+// GetPeriodsStartAndEndUTC returns the start and end timestamps (in milliseconds) given two months for location UTC (timezone)
+func GetPeriodsStartAndEndUTC(firstyyyymm string, lastyyyymm string) (start int64, end int64, err error) {
+	return getPeriodsStartAndEnd(firstyyyymm, lastyyyymm, *time.UTC)
+}
+
+// getPeriodsStartAndEnd returns the period start and end timestamps (in milliseconds) given two months and the location (timezone)
+func getPeriodsStartAndEnd(firstyyyymm string, lastyyyymm string, location time.Location) (start int64, end int64, err error) {
```

Comment:
unnecessary function, this code could be executed directly under `GetPeriodsStartAndEndUTC`
It is always time.UTC. Or should this function be exported?